pkg/operator/ceph/object/spec_test.go

```diff
@@ -194,10 +194,10 @@ func TestValidateSpec(t *testing.T) {
 	err = r.validateStore(s)
 	assert.NoError(t, err)
 
-	// external with no endpoints, failure
+	// external with no endpoints, success
```

Comment:
L192 to L200 makes no sense now, so I'd just remove both tests.

Reply:
Indeed...